src/2.0/schema/schema.json

```diff
@@ -142,7 +147,12 @@
               "properties": {
                 "type": {
                   "type": "string",
-                  "enum": ["NONE"]
+                  "enum": ["NONE", "REVOCATION_STORE"]
+                },
+                "address": {
```

Comment:
Might want to generalize this, we might have other methods like certificate revocation list etc in the future.

Reply:
what do they look like? could you give examples so I can see how I can generalise this?

Reply:
will change to a more generic key to `location` as discussed with raymond